lib/primitives/base-element.js

```diff
@@ -132,10 +165,22 @@ module.exports = function(registry) {
     },
 
     fromCompactRefract: function(tuple) {
-      this.meta = tuple[1];
       this.attributes = tuple[2];
       this.content = tuple[3];
 
+      if (tuple[1]) {
+        Object.keys(tuple[1]).forEach(function (key) {
+          var value = tuple[1][key];
+
+          // TODO: Warning - this may raise a false positive :-(
+          if (value && value.length === 4 && _.isString(value[0]) && _.isObject(value[1]) && _.isObject(value[2])) {
```

Comment:
This one concerns me more than the full serialization because it will create a broader signature for Refract. This will allow an array of four objects to be considered to be Compact Refract. I wonder if we could leave this one out for the time being.